src/ManojN/Q5_second_Largest_in_Array.java

```diff
@@ -0,0 +1,33 @@
+//5)Write a java program to find second largest element in an array of integers?
+package ManojN;
+
+public class Q5_second_Largest_in_Array {
+
+	
+	static void m1(int a2[]) {
+		// int a2[] = { 2, 6, 9, 3, 11 };
```

Comment:
Hi @ManojNAug19 
Here int a2[] = { 12,2, 6, 9, 3, 11 };
it print wrong output....

Debug your code...